README.md

```diff
@@ -1,2 +1,33 @@
 # Apple-website-webpage-clone
 This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.
+
+
+[screenshot](screenshot.PNG)
+
+## Live Demo
+---
+[Apple-website-webpage-clone] https://ecstatic-lamarr-cd95c7.netlify.app/
+
+## Built With
+---
+* CSS
+* HTML
+* Font awesome
+
+## Authors
+---
+#### Dibyendu Das
+* Github:[@DibDas](https://github.com/dibdas)
+* Twitter:[@DIBYEND78120480]
+* LinkedIn:[Dibyendu Das](https://www.linkedin.com/in/dibyendu-das-b5967a1b1/)
+
+#### Rayan Rassam
+* Github:[@Rayan84](https://github.com/Rayan84)
+* Twitter:[@Rayan65096267]https://twitter.com/Rayan65096267
```

Comment:
- [ ] Please fix the improper applied and broken links in your README to make your project look more professional.